src/js/themes/base.d.ts

```diff
@@ -1054,22 +1059,32 @@ export interface ThemeType {
     };
     critical?: {
       icon?: React.ReactNode;
+      background?: BackgroundType;
       color?: ColorType;
```

Comment:
What about `theme.notification.critical.toast.background` and `theme.notification.global.background`?

Reply:
Updated.